multicluster/controllers/multicluster/gateway_controller.go

```diff
@@ -212,6 +216,7 @@ func (r *GatewayReconciler) createResourceExport(ctx context.Context, req ctrl.R
 	resExportSpec.ClusterInfo = &mcsv1alpha1.ClusterInfo{
 		ClusterID:   r.localClusterID,
 		ServiceCIDR: r.serviceCIDR,
+		PodCIDRs:    r.podCIDRs,
```

Comment:
Question - if user changes the PodCIDRs option value and restarts controller, will the exported CIDRs be updated?

Reply:
It won't be updated unless we recreate the Gateway. Maybe we can add the "PodCIDRs" in the `NodeReconciler`, then update the Gateway to trigger the Gateway Reconcile then update the ClusterInfoExport?

Reply:
After controller restarts, anyway there will be a Gateway add event right, in which we should handle any change including clusterID, serviceCIDR, podCIDRs? @luolanzone

Reply:
yes, if controller restarts, there will be Gateway add event anyway. If user pass the PodCIDR through config file, the Gateway should be updated as well after controller restarts.

Reply:
> It won't be updated unless we recreate the Gateway. Maybe we can add the "PodCIDRs" in the `NodeReconciler`, then update the Gateway to trigger the Gateway Reconcile then update the ClusterInfoExport?

@jianjuns It will update, it's my mistake. When I test this case I didn't use the correct image.  When the Pod restart and ConfigMap changes, the ClusterInfoImport will be changed, too.